FluentUI/src/main/java/com/microsoft/fluentui/contextualcommandbar/CommandItemAdapter.kt

```diff
@@ -0,0 +1,170 @@
+package com.microsoft.fluentui.contextualcommandbar
+
+import android.content.Context
+import android.support.v4.content.ContextCompat
+import android.support.v7.content.res.AppCompatResources
+import android.support.v7.widget.RecyclerView
+import android.view.View
+import android.view.ViewGroup
+import android.view.ViewGroup.LayoutParams.WRAP_CONTENT
+import android.widget.ImageView
+import com.microsoft.fluentui.R
+
+class CommandItemAdapter(val context: Context) : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
```

Comment:
Interestingly because the Adapter is needed in order to set the click event, this requires the consumer to add a dependency on RecyclerView, which may or may not be desired. I am not sure if we want to keep that internal and instead allow the consumer to set some more generic event?

Reply:
Removed the click listener to CommandItem